### ISSUE TITLE
¿como agregar nueva fuente?

### ISSUE THREAD

**Issue by Lucasmercado** [google/fonts#1833]
Hola, buenas tardes! soy nuevo por acá, quisiera contribuir con una nueva fuente a google fonts ¿como puedo agregarla? yo soy el diseñador de la fuente, pueden verla en el siguiente post: https://www.behance.net/gallery/74930547/Chappa-Free-Font-typeface

**Comment by laerm0** [comment]
Hola. We have a [guide for contributing](https://github.com/google/fonts/blob/master/CONTRIBUTING.md) set up, so please take a look and let us know if you have any questions.

**Comment by Lucasmercado** [comment]
> Hola. We have a [guide for contributing](https://github.com/google/fonts/blob/master/CONTRIBUTING.md) set up, so please take a look and let us know if you have any questions.

Sí, leí la guía de contribución y cumplo con los requisitos. Mi pregunta es ¿cuáles son los pasos para que la fuente pueda publicarse en Google fonts? No soy programador ¿es necesario?, gracias!

**Comment by laerm0** [comment]
Hmm, give it a shot @felipesanches

**Comment by felipesanches** [comment]
Oh! And, by the way, I liked the font. Pretty cool :-D

<img width="954" alt="captura de tela 2019-01-30 as 02 41 03" src="https://user-images.githubusercontent.com/213676/51958785-d877a900-2438-11e9-8658-d1f1f6f5a0f1.png">

**Comment by Lucasmercado** [comment]
> @laerm0 he said he is not a programmer and asked if it would be a prerequisite to contribute a font.
> 
> @Lucasmercado you don't need to be a programmer. Having created a font that you are willing to share under the Open Font License (OFL) is enough. You would also need to publish the Fontlab source files.
> 
> I think @davelab6 may be able to provide further help.

Muchas gracias felipe :)

**Comment by felipesanches** [comment]
hi, @Lucasmercado. I tried to access the link to your font page on Behance again today (https://www.behance.net/gallery/74930547/Chappa-Free-Font-typeface) but it seems to be offline.

Searching on Google I found this other page:
https://pinspiry.com/chappa-free-font/

After downloading the zip file, I saw in the readme.txt that you licensed it under "Creative Commons Attribution-NoDerivatives 4.0". In order to contribute the font to the Google Fonts collection you would have to change the licensing to use the SIL Open Font License. Would you like to do so?

**Comment by Lucasmercado** [comment]
Hola, @felipesanches. Si puedo cambiar la licencia.

**Comment by davelab6** [comment]
@Lucasmercado we are finally getting around to processing the many proposals to Google Fonts, like yours. Which font editor software did you use to create it? :) We can help you get the project ready if you have the source files available. And perhaps since 2019 you may have learned more about how to use Github? :) If no, no problem, you can just email the file and we can take it from there.
